spec/index.bs

```diff
@@ -4495,6 +4502,10 @@ enum GPUTextureAspect {
                                 |this|.{{GPUTexture/format}},
                                 |descriptor|.{{GPUTextureViewDescriptor/aspect}}).
 
+                        - |descriptor|.{{GPUTextureViewDescriptor/usage}} must be a subset of |this|.{{GPUTexture/usage}}.
+                        - If |descriptor|.{{GPUTextureViewDescriptor/usage}} includes the {{GPUTextureUsage/STORAGE_BINDING}} bit:
+                            - |descriptor|.{{GPUTextureViewDescriptor/format}} must be listed in [[#plain-color-formats]] table
+                                with {{GPUTextureUsage/STORAGE_BINDING}} capability for the appropriate access mode.
```

Comment:
What is the appropriate access mode? I don't think we know at `createView()` time. I think it just needs to be for _any_ access mode?

(The rule in `createBindGroup()` still checks the access mode:

> entry.storageTexture.format must be a format which can support storage usage for the given entry.storageTexture.access according to the § 26.1.1 Plain color formats table.

)

Reply:
Changed to "for at least one access mode"

This text was lifted from `validating GPUTextureDescriptor`, so I fixed it in both places.